.github/workflows/if-nodejs-release.yml

```diff
@@ -38,24 +38,24 @@ jobs:
           git config --global core.autocrlf false
           git config --global core.eol lf
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Check if Node.js project and has package.json
         id: packagejson
-        run: test -e ./package.json && echo "::set-output name=exists::true" || echo "::set-output name=exists::false"
+        run: test -e ./package.json && echo "exists=true" >> $GITHUB_OUTPUT || echo "exists=false" >> $GITHUB_OUTPUT
         shell: bash
       - if: steps.packagejson.outputs.exists == 'true'
         name: Setup Node.js
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
-          node-version: 14
+          node-version: 18
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
       - if: steps.packagejson.outputs.exists == 'true'
         name: Install dependencies
         run: npm install
       - if: steps.packagejson.outputs.exists == 'true'
         name: Run test
-        run: npm test
+        run: npm test --if-present
```

Comment:
Same as above. It should become `npm test --if-present --workspaces`.

Reply:
I don't think we need the --workspaces flag here. as you can see in this run, everything works fine without it. https://github.com/KhudaDad414/studio/actions/runs/5333204486/workflow
it's the job of turborepo to handle workspaces here.

Reply:
I even moved the `craco` package back to the `studio` app and everything seems to be working fine. The problem was with the version 14 of node that we were using. [turborepo min is 16](https://github.com/vercel/turbo/issues/2336) and we are using `tsup` which requires node 18.